app/onboarding/page.tsx

```diff
@@ -0,0 +1,179 @@
+'use client';
+
+import React, { useState } from 'react';
+import { upsertProfile } from '@/api/supabase/queries/profiles';
+import { Profile } from '@/types/schema';
+
+type UUID = `${string}-${string}-${string}-${string}-${string}`;
+const generateUUID = (): UUID => {
+  return crypto.randomUUID() as UUID;
+};
+const id = generateUUID();
```

Comment:
I believe we've configured the `profiles` table so that it generates a UUID for you whenever you insert into the table, so we don't have to do it ourselves

Reply:
Noted: Used a UUID from Auth